app/src/main/java/com/android/streetworkapp/ui/profile/Profile.kt

```diff
@@ -23,66 +24,56 @@ import androidx.compose.ui.unit.sp
+  Box(modifier = Modifier.testTag("ProfileScreen")) {
+    Column(
+        modifier = Modifier.fillMaxSize().padding(innerPaddingValues).testTag("ProfileColumn"),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.spacedBy(10.dp, Alignment.CenterVertically)) {
+          // profile placeholder
 
-              Image(
-                  painter = painterResource(id = R.drawable.profile),
-                  contentDescription = "profile picture",
-                  modifier = Modifier.size(200.dp))
+          Image(
+              painter = painterResource(id = R.drawable.profile),
+              contentDescription = "profile picture",
+              modifier = Modifier.size(200.dp))
 
-              // score placeholder
+          // score placeholder
 
-              Text(
-                  text = "Score: 42’424",
-                  fontSize = 20.sp,
-                  modifier = Modifier.testTag("profileScore"))
+          Text(
+              text = "Score: 42’424", fontSize = 20.sp, modifier = Modifier.testTag("profileScore"))
 
-              // QR code placeholder
+          // QR code placeholder
 
-              Image(
-                  painter = painterResource(id = R.drawable.qrcode),
-                  contentDescription = "qr code",
-                  modifier = Modifier.size(260.dp))
+          Image(
+              painter = painterResource(id = R.drawable.qrcode),
+              contentDescription = "qr code",
+              modifier = Modifier.size(260.dp))
 
-              Spacer(modifier = Modifier.height(10.dp))
+          Spacer(modifier = Modifier.height(10.dp))
 
-              // button to go to add friend screen
-              Button(
-                  onClick = { navigationActions.navigateTo(Screen.ADD_FRIEND) },
-                  modifier = Modifier.size(220.dp, 50.dp).testTag("profileAddButton")) {
-                    Text(text = "Add a new friend", fontSize = 17.sp)
-                  }
+          // button to go to add friend screen
+          Button(
+              onClick = { navigationActions.navigateTo(Screen.ADD_FRIEND) },
+              modifier = Modifier.size(220.dp, 50.dp).testTag("profileAddButton")) {
+                Text(text = "Add a new friend", fontSize = 17.sp)
+              }
 
-              Spacer(modifier = Modifier.height(10.dp))
-              // button to train with a friend
-              Button(
-                  onClick = {},
-                  colors = ButtonDefaults.buttonColors(Color(0xFFA53A36)),
-                  modifier = Modifier.size(220.dp, 50.dp).testTag("profileTrainButton")) {
-                    Text(text = "Train with a friend", fontSize = 17.sp)
-                  }
-            }
-      })
+          Spacer(modifier = Modifier.height(10.dp))
+          // button to train with a friend
+          Button(
+              onClick = {},
+              colors = ButtonDefaults.buttonColors(Color(0xFFA53A36)),
```

Comment:
Improvement: probably not in your task range but we should avoid setting directly a color instead use the palette of the `Color.kt`

Reply:
Since we'll have to rework the Profile screen anyways, it might be a good idea to do it then

Reply:
Yeah I agree